dev/devicelab/manifest.yaml

```diff
@@ -505,7 +505,9 @@ tasks:
     description: >
       Runs benchmarks from dev/benchmarks/microbenchmarks on iPhone 6.
     stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios"]
+    #TODO(godofredoc): Remove ios/12 capability once
```

Comment:
When you do remove this, make sure https://github.com/flutter/flutter/issues/49504 is fixed before you go past 13.3.1.

Reply:
Actually never mind, it's fine as long as we're signing with the org provisioning profiles.